src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier supplier = new ColorSupplier();
+    private final Random randomizer = new Random();
+    public static final int MAX_NUMBER = 100;
```

Comment:
```suggestion
    private static final int MAX_NUMBER = 100;
```
Also, let’s follow the [ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents)